### PR TITLE
Manage temp folder

### PR DESCRIPTION
**Pull request by cfelde**

### CODE REVIEW

**Comment by cfelde** [comment]
Build folder is often cleaned, like when doing ./gradlew clean, so think it should be fine.

File: src/test/java/org/web3j/console/project/kotlin/KotlinProjectImporterTest.java [hunk 1]
```diff
@@ -12,41 +12,32 @@
  */
 package org.web3j.console.project.kotlin;
 
-import java.io.ByteArrayOutputStream;
-import java.io.File;
-import java.io.IOException;
-import java.io.PrintStream;
-import java.nio.file.Path;
+import java.io.*;
```
Comment (iikirilov) [inline src/test/java/org/web3j/console/project/kotlin/KotlinProjectImporterTest.java]: nit: remove wildcard

File: src/test/java/org/web3j/console/KeyImporterTest.java [hunk 2]
```diff
@@ -30,26 +29,27 @@
     IODevice console = mock(IODevice.class);
 
     @Test
-    public void testSpecifyPrivateKey(@TempDir Path tempDirPath) {
-        prepareWalletCreation(SampleKeys.PRIVATE_KEY_STRING, tempDirPath);
+    public void testSpecifyPrivateKey() {
```
Comment (iikirilov) [inline src/test/java/org/web3j/console/KeyImporterTest.java]: So the core of the issue using this @TempDir annotation?

Comment (cfelde) [reply src/test/java/org/web3j/console/KeyImporterTest.java]: Mac OS security model likely core issue

**Pull request merged**